name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/actions/InsertAction.java

```diff
@@ -44,6 +45,11 @@ public void setInvestmentPlanItem(boolean flag)
         this.investmentPlanItem = flag;
     }
 
+    public void setRemoveNotesInTransactions(boolean flag)
+    {
+        this.addNotesInTransactions = flag;
+    }
+
```

Comment:
This is the other place where the meaning is flipped: the method says it configures whether to remove notes, but the internal flag then is then whether to add notes.